release.sh

```diff
@@ -71,7 +71,14 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+until docker images | grep "containers.schibsted.io/finntech/node" | awk '{print $3}' | sort | uniq | xargs docker rmi -f
+do
+  true
+done
```

Comment:
This is better written as `docker images |  awk '/containers.schibsted.io/ { print $3 }' | sort -u | xargs --no-run-if-empty docker rmi -f`